contrib/remote-dev/README.md

```diff
@@ -0,0 +1,165 @@
+# Remote Development
+
+## 1. Overview
+
+Remote Development can allow users to develop locally with PAI's resources and improve users’ development experience.
+
+Basicaly, the main process is to ask PAI for a container with resources, and use local editors or IDEs to develope and debug in this container using ssh.
+
+Support IDEs or editors:
+- CLI
+- Visual Studio Code
+- PyCharm
+
+Support OS:
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Windows 10
+
+***NOTICE: This is still an experiment solution and may be changed in future release.***
+
+## 2. Directory Structure
+
+```
+remote-dev/
+├── README.md
+├── conf
+│   ├── clusters.template        # PAI cluster template conf
+│   └── job.template             # PAI template job
+└── start.py                     # start script
+```
+
+## 3. Dependencies
+
+- This sub project depend on [PAI's Python SDK](https://github.com/microsoft/pai/tree/master/contrib/python-sdk), please install and test the sdk first.
+
+- The `start.py` script requires python3, and we only tested it on `py3.5+` environment, and ruquired following moduls:
+    - requests
+    - configparser
+    - subprocess
+
+## 3. Usage
+
+This section will give guidance about usage.
+
+### 3.1 Preparation
+
+First, please configure the vars in `.env.template` and rename it to `.env`.
+
+```
+[PAI_ENV]
+username=                        # PAI cluster user name
+password=                        # PAI cluster password
+serverip=                        # PAI cluster ip
+```
+
+After configuration, you should edit the job template file named `./conf/job.template`.
+
+The original job template file is like:
+
+```yaml
+protocolVersion: 2
+name: remote_dev_job_template
+type: job
+jobRetryCount: 0
+prerequisites:
+  - type: dockerimage
+    uri: openpai/pai.example.tensorflow
+    name: docker_image_0
+taskRoles:
+  Task_role_1:
+    instances: 1
+    completion:
+      minFailedInstances: 1
+      minSucceededInstances: 1
+    dockerImage: docker_image_0
+    resourcePerInstance:
+      gpu: 0
+      cpu: 1
+      memoryMB: 2048
+    commands:
+      - '`#REMOTE_DEV_START`'
+      - apt-get update
+      - apt-get install -y nfs-common cifs-utils sshpass wget
+      - umask 000
+      - mkdir -p /root/workspace
+      - mount.cifs //sharepath /root/workspace -o username=***,password=***
```

Comment:
Do we need to change set the value for username & password. Or we just leave it as `***`?

Reply:
It should be changed. I will make this more clear.